### ISSUE TITLE
decryptFail not found in normative section of symmetric spec

### ISSUE THREAD

**Issue by atvassilev** [usnistgov/ACVP#141]
decryptFail only shows in the example in Appendix D. It needs to be defined in Section 4.

**Comment by Kritner** [comment]
@atvassilev I noticed it in @bfussell's last email the in example:

```
            {
                "tcId": 538,
                "decryptFail": true
            },
```
The `true` was wrapped in quotes, which would be interpreted as a string, rather than the intended boolean.

Could that also be remedied in this issue as well?  Booleans should not be wrapped in quotes, so it would look like:

```
            {
                "tcId": 538,
                "decryptFail": "true"
            },
```

**Comment by bfussell** [comment]
will do

**Comment by atvassilev** [comment]
Resolved with PR #142.
